### ISSUE TITLE
Deploy FERC XBRL derived data to Datasette

### ISSUE THREAD

**Issue by zaneselvans** [catalyst-cooperative/pudl#2080]
Our pipeline now extracts FERC data from XBRL for Forms 1, 2, 6, 60, and 714 and writes it to SQLite databases.

Add logic to ensure that those DBs can be deployed to our Datasette instance at http://data.catalyst.coop in the same way that the current PUDL and DBF derived FERC Form 1 data are.

* [x] Update the Datasette deployment script to deploy the XBRL derived SQLite DBs which are built by the nightly builds already.
* [x] Fix `pudl.metadata.DatasetteMetadata` class to generate separate annotations for each of the DBs. Currently all of the XBRL DB metadata appears to be combined with the PUDL DB metadata, while the FERC Form 1 DBF data is (appropriately) separate.

**Comment by zaneselvans** [comment]
@zschira messing around with this a bit locally, it looks like the wonky metadata output doesn't cause the deployment to fail. Using this command from within the `devtools/datasette` directory should start up a local server for inspection:

```bash
datasette_metadata_to_yml -o "metadata.yml"
datasette serve -m metadata.yml ../../../pudl-work/sqlite/{pudl,ferc1,ferc1_xbrl,ferc2_xbrl,ferc6_xbrl,ferc60_xbrl,ferc714_xbrl}.sqlite
```

The metadata for the XBRL databases just doesn't show up on the site, so they'll be hard to use until we get that sorted out. But they are there and browsable, so I think I'll go ahead and add them to the deployment script (after testing that the deployment works by hand locally...)

**Comment by zschira** [comment]
This ended up requiring minor changes to the `DatasetteMetadata` class as well as changes to the jinja template, but I can now deploy datasette locally with the metadata showing up properly.

**Comment by zaneselvans** [comment]
There's a Datasette metadata parsing test which looks like it's failing for me locally:
```
 pytest test/integration/datasette_metadata_test.py
```

**Comment by zaneselvans** [comment]
Oh nevermind this is a simple thing. Just failing because there are new databases now.

**Comment by zaneselvans** [comment]
Closed by PR #2095
